src/java/us/kbase/mobu/ModuleBuilder.java

```diff
@@ -361,7 +360,22 @@ private static int runTestCommand(TestCommandArgs testArgs, JCommander jc) {
 
     private static void printVersion() {
         String gitCommit = getGitCommit();
-        System.out.println("KBase SDK version " + VERSION + (gitCommit == null ? "" : (" (commit " + gitCommit + ")")));
+        String timestamp = getBuildTimestamp();
+        System.out.println("KBase SDK version " + VERSION
+            + (timestamp == null ? "" : ("_" + timestamp))
+            + (gitCommit == null ? "" : (" (commit " + gitCommit + ")")));
+    }
+
+    public static String getBuildTimestamp() {
+        String buildTimestamp = null;
+        try {
+            Properties gitProps = new Properties();
+            InputStream is = ModuleBuilder.class.getResourceAsStream("git.properties");
+            gitProps.load(is);
+            is.close();
+            buildTimestamp = gitProps.getProperty("build_timestamp");
+        } catch (Exception ignore) {}
```

Comment:
IMO if an exception occurs here the process should fail, as it means the `build.xml` file is borked and thus the build is borked.

Also this is almost completely duplicated by `getGitCommit()` below, DRYification is possible.

Reply:
👍